analyzer-models/src/main/java/ffb/analyzer/models/espn/LeagueInformation.java

```diff
@@ -0,0 +1,212 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import ffb.analyzer.models.espn.deserializers.EpochMillisecondDeserializer;
+import ffb.analyzer.models.espn.deserializers.IsoDateWithOffsetDeserializer;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Entity representing information about a Fantasy Football league.
+ */
+public class LeagueInformation extends EspnEntity<LeagueInformation> {
+
+    public enum LeagueType {
+        PRIVATE(0),
+        PUBLIC(1);
+
+        LeagueType(int type) {
+        }
+    }
+
+    private boolean isFull;
+    private boolean isActive;
+    private boolean isExpired;
+    private boolean isWillBeDeleted;
```

Comment:
`willBeDeleted`?

Reply:
I think we should try to stick to a single leading string for boolean properties/methods. Or we could potentially reserve one for props and one for methods? I think "is" would be the most versatile. We could make this property isToBeDeleted instead.